sdk/src/driftClient.ts

```diff
@@ -271,7 +269,7 @@ export class DriftClient {
 	): User {
 		const userAccountPublicKey = getUserAccountPublicKeySync(
 			this.program.programId,
-			authority ?? this.authority,
+			authority ?? this.wallet.publicKey,
```

Comment:
i think we still want this to be this.authority (and all the other places this pr switched  this.authority for this.wallet.publicKey